pkg/scheduler/cache/cache.go

```diff
@@ -1225,8 +1226,21 @@ func (sc *SchedulerCache) GetMetricsData() {
 				if len(warnings) > 0 {
 					klog.V(3).Infof("Warning querying Prometheus: %v", warnings)
 				}
+				if res == nil || res.String() == "" {
+					klog.V(3).Infof("Warning querying Prometheus: no data found for %s", queryStr)
```

Comment:
Suggest to make use of `klog.warning` for this process.

Reply:
done